lms/templates/admin/instance.html.jinja2

```diff
@@ -79,7 +79,12 @@
     <fieldset class="box">
         <legend class="label has-text-centered">Canvas settings</legend>
         {{ macros.disabled_text_field("API domain", instance.custom_canvas_api_domain) }}
+
         {{ macros.disabled_text_field("Developer key", instance.developer_key) }}
+        <div class="has-text-right mb-6">
```

Comment:
Don't try to upgrade this in-place, we avoid having to worry about the secret value.